packages/flutter_adaptive_scaffold/pubspec.yaml

```diff
@@ -1,8 +1,8 @@
-name: adaptive_scaffold
+name: flutter_adaptive_scaffold
 description: Widgets to easily build adaptive layouts, including navigation elements.
 version: 0.0.3
-issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+adaptive_scaffold%22
-repository: https://github.com/flutter/packages/tree/main/packages/adaptive_scaffold
+issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+flutter_adaptive_scaffold%22
```

Comment:
The label "flutter_adaptive_scaffold" does exit in the Flutter repository. I created a ticket to rename the "adaptive_scaffold" label to "flutter_adaptive_scaffold": https://github.com/flutter/flutter/issues/111832